app/views/cookies/cookie-banner.njk

```diff
@@ -0,0 +1,85 @@
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+
+<noscript>
+{{ govukCookieBanner({
+  ariaLabel: "Cookies on Calculate my progressive reductions",
+  messages: [
+    {
+      headingText: "Cookies on Calculate my progressive reductions",
+      text: "We use cookies to make this service work and collect analytics information. To accept or reject cookies, turn on JavaScript in your browser settings or reload this page."
+    }
+  ]
+}) }}
+</noscript>
+
+<div class="js-cookies-container" style="display:none">
+  {% set html %}
+    <p>We use some essential cookies to make this service work.</p>
+    <p>We’d also like to use analytics cookies so we can understand how you use the service and make improvements.</p>
+  {% endset %}
+
+  {% set acceptHtml %}
+    <p>You’ve accepted analytics cookies. You can <a href="/cookies">change your cookie settings</a> at any time.</p>
+  {% endset %}
+
+  {% set rejectHtml %}
+    <p>You’ve rejected analytics cookies. You can <a href="/cookies">change your cookie settings</a> at any time.</p>
+  {% endset %}
+
+  {{ govukCookieBanner({
+    ariaLabel: "Cookies on Calculate my progressive reductions",
+    classes: "js-cookies-banner",
+    messages: [
+      {
+        headingText: "Cookies on Calculate my progressive reductions",
+        html: html,
+        classes: "js-question-banner",
+        actions: [
+          {
+            text: "Accept analytics cookies",
+            type: "button",
+            name: "cookies",
+            value: "accept",
+            classes: "js-cookies-button-accept"
+          },
+          {
+            text: "Reject analytics cookies",
+            type: "button",
+            name: "cookies",
+            value: "reject",
+            classes: "js-cookies-button-reject"
+          },
+          {
+            text: "View cookies",
+            href: "/cookies"
+          }
+        ]
+      },{
+        html: acceptHtml,
+        role: "alert",
+        hidden: true,
+        classes: "js-cookies-accepted",
+        actions: [
+          {
+            text: "Hide this message",
+            classes: "js-hide"
+          }
+        ]
+      },
+      {
+        html: rejectHtml,
+        role: "alert",
+        hidden: true,
+        classes: "js-cookies-rejected",
+        actions: [
+          {
+            text: "Hide this message",
+            classes: "js-hide"
+          }
+        ]
+      }
+    ]
+  }) }}
+</div>
+
+<script src="/static/scripts/cookies.js"></script>
```

Comment:
This page should be a template with webpack and the `cookies.js` file should be in `frontend/js/` like in the map application.

Reply:
Page should be renamed `_cookies-banner.template.njk`